src/main/java/frc/robot/subsystems/Arm.java

```diff
@@ -108,6 +109,15 @@ public void periodic() {
         Logger.getInstance().processInputs(getName(), data);
     }
 
+    private double softLimit(double speed) {
+        if (data.distanceInches > SOFT_MAX_DISTANCE && speed > 0) {
```

Comment:
Why are there two conditions that do the same thing?

Reply:
huh?